akka-stream-tests/src/test/scala/akka/stream/scaladsl/FlowConflateSpec.scala

```diff
@@ -168,6 +169,8 @@ class FlowConflateSpec extends AkkaSpec {
       sub.expectRequest(1)
       sub.sendNext(3)
 
+      sinkProbe.expectNoMsg(100.millis)
```

Comment:
add a comment that this is to avoid race between the above sendNext and the request below (fuzzer is fun ;) ).